include/cuda/std/cmath

```diff
@@ -9,19 +9,19 @@
 #ifndef _CUDA_CMATH
 #define _CUDA_CMATH
 
+#ifndef __CUDACC_RTC__
+#include <math.h>
+#include <cmath>
```

Comment:
I am considerably scared that we need both of them

Technically one should suffice

Reply:
Sorry, didn't mean to leave it in, we do only need host `<cmath>`.